demo-app/ng8/src/polyfills.ts

```diff
@@ -0,0 +1,74 @@
+ *   2. Application imports. Files imported after ZoneJS that should be loaded before your main
+ *      file.
+ *
+ * The current setup is for so-called "evergreen" browsers; the last versions of browsers that
+ * automatically update themselves. This includes Safari >= 10, Chrome >= 55 (including Opera),
+ * Edge >= 13 on the desktop, and iOS 10 and Chrome on mobile.
+ *
+ * Learn more in https://angular.io/guide/browser-support
+ */
+
+/***************************************************************************************************
+ * BROWSER POLYFILLS
+ */
+/* tslint:disable:no-import-side-effect */
+/**
+ * IE9, IE10 and IE11 requires all of the following polyfills.
+ */
+import "core-js/es";
+
+/**
+ * IE10 and IE11 requires the following for NgClass support on SVG elements
+ */
+import "eligrey-classlist-js-polyfill";
+
+/**
+ * IE10 and IE11 requires the following for the Reflect API.
+ */
+import "core-js/proposals/reflect-metadata";
+
+/**
+ * Web Animations `@angular/platform-browser/animations`
+ * Only required if AnimationBuilder is used within the application and using IE/Edge or Safari.
+ * Standard animation support in Angular DOES NOT require any polyfills (as of Angular 6.0).
+ */
+// import "web-animations-js";  // Run `npm install --save web-animations-js`.
+
+/**
+ * By default, zone.js will patch all possible macroTask and DomEvents
+ * user can disable parts of macroTask/DomEvents patch by setting following flags
+ * because those flags need to be set before `zone.js` being loaded, and webpack
+ * will put import in the top of bundle, so user need to create a separate file
+ * in this directory (for example: zone-flags.ts), and put the following flags
+ * into that file, and then add the following code before importing zone.js.
+ * import './zone-flags.ts';
+ *
+ * The flags allowed in zone-flags.ts are listed here.
+ *
+ * The following flags will work for all browsers.
+ *
+ * (window as any).__Zone_disable_requestAnimationFrame = true; // disable patch requestAnimationFrame
+ * (window as any).__Zone_disable_on_property = true; // disable patch onProperty such as onclick
+ * (window as any).__zone_symbol__UNPATCHED_EVENTS = ["scroll", "mousemove"]; // disable patch specified eventNames
+ *
+ * in IE/Edge developer tools, the addEventListener will also be wrapped by zone.js
+ * with the following flag, it will bypass `zone.js` patch for IE/Edge
+ */
+(window as any).__Zone_enable_cross_context_check = true;
+
+/***************************************************************************************************
+ * Zone JS is required by default for Angular itself.
+ */
+import "zone.js/dist/zone"; // Included with Angular CLI.
+
+/***************************************************************************************************
+ * APPLICATION IMPORTS
+ */
+// FixMe: Remove when NBB upgrades chrome browser (https://caniuse.com/#search=grid)
+import "./polyfills/css-grid-layout-polyfill";
```

Comment:
Should we remove this import, and the file "css-grid-layout-polyfill", since we are using Chrome >v74 at NBB ? 😊 
Or should we keep it for another reason?

Reply:
Ah true, we should remove it. I'll remove it and test it also in IE 😉

Reply:
I've tested and indeed, we don't need the polyfill for Chrome, however it is necessary for IE but... the problem in IE with this polyfill is that it causes some flickering which is quite annoying.

So I think we should just remove the polyfill and let IE display the grid the best it can... which doesn't look too bad in my opinion, you know... it's IE 😕